src/gluonts/torch/scaler.py

```diff
@@ -141,20 +130,11 @@ class StdScaler(nn.Module):
         along dimension ``dim``.
     """
 
-    @validated()
-    def __init__(
-        self, dim: int = -1, keepdim: bool = False, minimum_scale: float = 1e-5
-    ):
-        super().__init__()
-        assert dim > 0, (
-            "Cannot compute scale along dim = 0 (batch dimension), please"
-            " provide dim > 0"
-        )
-        self.dim = dim
-        self.keepdim = keepdim
-        self.register_buffer("minimum_scale", torch.tensor(minimum_scale))
+    dim: int = -1
+    keepdim: bool = False
+    minimum_scale: float = 1e-5
```

Comment:
Also, `minimum_scale` should be a `torch.Tensor`.

Reply:
Why? I think we can add a `torch.Tensor` to a number